### DISCUSSION TITLE
OpenAI SDK wrapper for JS

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#1065] [Ideas]
For users who just want to log OpenAI calls to Langfuse, getting started with Langfuse could be as simple as:

```typescript
// openai
import OpenAI from 'openai';

const openai = new OpenAI({
  apiKey: 'my api key',
});

// openai with Langfuse
import OpenAI from 'langfuse-openai';

const openai = new OpenAI({
  apiKey: 'my api key',
  langfuseSecretKey: 'lf-sk-...',
  langfusePublicKey: 'lf-pk-...',
});
```

Similar to the existing Python integration: https://langfuse.com/docs/openai

**Comment by kaaancan** [comment]
This is exactly my usecase. Right now i have my own database tables where i need to manually add every place i do OpenAI calls.

This featire would be really appreciated !

**Comment by marcklingen** [comment]
+1 by @louis030195 in #1306

**Comment by hassiebp** [comment]
@kaaancan We've just released the JS integration for the OpenAI SDK in the [Langfuse JS SDK 3.6.0](https://github.com/langfuse/langfuse-js/releases/tag/v3.6.0) 🎉 Feel free to try it out and let us know your feedback 🙏🏾

[Link to release](https://github.com/langfuse/langfuse-js/releases/tag/v3.6.0)
[Link to docs](https://langfuse.com/docs/integrations/openai/js/get-started)
